bridges/SteamBridge.php

```diff
@@ -117,34 +70,47 @@ public function collectData(){
 				}
 			}
 
+			$coverUrl = str_replace('_292x136', '', strtok($element->capsule, '?'));
+			$picturesPath = pathinfo($coverUrl)['dirname'] . '/';
+
 			$item = array();
 			$item['uri'] = "http://store.steampowered.com/app/$id/";
 			$item['title'] = $element->name;
 			$item['type'] = $appType;
-			$item['cover'] = str_replace('_292x136', '', $element->capsule);
+			$item['cover'] = $coverUrl;
 			$item['timestamp'] = $element->added;
 			$item['isBuyable'] = $appIsBuyable;
 			$item['hasDiscount'] = $appHasDiscount;
 			$item['isFree'] = $appIsFree;
 			$item['priority'] = $element->priority;
 
 			if($appIsBuyable) {
+
 				$item['price'] = floatval(str_replace(',', '.', $appPrice));
+				$item['content'] = $appPrice;
+
+			}
+
+			if($appIsFree) {
+				$item['content'] = 'Free';
 			}
 
 			if($appHasDiscount) {
 
 				$item['discount']['value'] = $appDiscountValue;
-				$item['discount']['oldPrice'] = floatval(str_replace(',', '.', $appOldPrice));
-				$item['discount']['newPrice'] = floatval(str_replace(',', '.', $appNewPrice));
+				$item['discount']['oldPrice'] = $appOldPrice;
+				$item['content'] = '<s>' . $appOldPrice . '</s> <b>' . $appPrice . '</b> (' . $appDiscountValue . ')';
 
 			}
 
 			$item['enclosures'] = array();
-			$item['enclosures'][] = str_replace('_292x136', '', $element->capsule);
+			$item['enclosures'][] = $coverUrl;
+
+			foreach($element->screenshots as $screenshotFileName) {
+
+				$screenshotUrl = $picturesPath . $screenshotFileName;
+				$item['enclosures'][] = $screenshotUrl;
```

Comment:
I think this can be done in one step (without `$screenshotUrl`)

```suggestion
				$item['enclosures'][] = $picturesPath . $screenshotFileName;
```